src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1975,6 +1977,11 @@ void PgScrubber::dump_scrubber(ceph::Formatter* f,
     f->dump_string("schedule", sched_state);
   }
 
+  if (m_publish_sessions) {
+    f->dump_int("test_sequence",
+                m_sessions_counter);  // an ever-increasing number used by tests
```

Comment:
This implementation (which does not affect the user output) is much better.  IIUC, the motivation behind adding this, is to have a simple way to check for scrub start/finish events without having to scrape through timestamps/logs etc.?

Reply:
Yes. And I plan to change most tests to use this mechanism (thus shortening the tests).
Didn't want to do that in this specific PR.